src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -219,11 +232,9 @@ def create(dataset, target, feature, max_iterations=10,
                                     label=train_data['labels'].to_numpy(),
                                     batch_size=training_batch_size, shuffle=True)
 
-    custom_NN = SoundClassifier._build_custom_neural_network(feature_extractor.output_length, num_labels, custom_layer_sizes)
+    from ._mx_sound_classifier import SoundClassifierMXNETModel
     ctx = _mxnet_utils.get_mxnet_context()
```

Comment:
Could we move the MXNet context into `SoundClassifierMXNETModel`, too? I think it's too tied to the NN training, as an implementation detail. There won't be any effective way to use this with TensorFlow.

Reply:
Done